jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -284,6 +293,26 @@ private void checkNonSunProxyMemberAccess(SecurityManager security, ClassLoader
 	}
 }
 
+/*
+ * This helper method atomically writes the given \p fieldvalue to the 
+ * field specified by the \p fieldname
+ */
+private long getFieldOffsetAndCacheFieldValue(String fieldname, Object fieldvalue){
```

Comment:
Please capitalize the first letter of each internal word so the parameters look like `fieldName` & `fieldValue`.